tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -371,6 +368,8 @@ build_rootfs_distro()
 		elif [ -n "${USE_PODMAN}" ]; then
 			container_engine="podman"
 			engine_build_args+=" --runtime ${DOCKER_RUNTIME}"
+		elif [ -n "${USE_NERDCTL}" ]; then
```

Comment:
Please can you add `USE_NERDCTl` in `tools/osbuilder/README.md` about how to use it and the use case, for example, combinate it with buildkit.

Reply:
And please add an option description to the help message https://github.com/kata-containers/kata-containers/blob/f1381eb361aa2202d48607df8cfc6dbfe6db580c/tools/osbuilder/rootfs-builder/rootfs.sh#L151-L157

Reply:
Sure. will add more information and an introduction for related docs within this PR. Besides, I feel it's better to split this PR into two one of which is only formatting the script.

Reply:
Yes please, this should be split into two separate commits. Please include all the formatting changes in a separate commit.